src/routes/edit/share-options/exports/frameworks/angular/fragment-v10.ts

```diff
@@ -75,6 +76,22 @@ const getComponentCode = (fragment: any, fragments: any[]) => {
 	return componentCode;
 };
 
+const checkForPlaceholder = (json: any) => {
```

Comment:
```suggestion
const hasPlaceholder = (json: any) => {
```
?

And it can be called in place perhaps, since it's only called once